File: PHP/Human.php
<?php
class Human
{
    // Membuat atribut Private dari kelas Human
    private $nama = '';
    private $nik = '';
    private $gender = '';
    private $image;

    # Constructor with default initialization
    public function __construct($nama = '', $nik = '', $gender = '',  $image)
    {
        $this->nama = $nama;
        $this->gender = $gender;
        $this->nik = $nik;
        $this->image = $image;
    }

    # Setter and Getter methods for private attributes

    # Set Nama
    public function setNama($namaMahasiswa)
    {
        $this->nama = $namaMahasiswa;
    }

    # Get Nama
    public function getNama()
    {
        return $this->nama;
    }

    # Set nik
    public function setNik($nik)
    {
        $this->nik = $nik;
    }

    # Get nik
    public function getNik()
    {
        return $this->nik;
    }

    # Set gender
    public function setGender($gender)
    {
        $this->gender = $gender;
    }

    # Get Gender
    public function getGender()
    {
        return $this->gender;
    }

    # Set Image
    public function setImage($image)
    {
        $this->image = $image;
    }

    # Get Image
    public function getImage()
    {
        return $this->image;
    }
}

File: Python/Human.py
# Mmebuat kelas HUman
class Human:
    # Membuat beberapa atirbut private
    # Nama Human
    __nama = ""
    # nik Human
    __nik = ""
    # jenis kelamin Human
    __Jenis_Kelamin = ""

    # Constructor with default initialization
    def __init__(self, nama="", nik="", Jenis_Kelamin=""):
        self.__nama = nama
        self.__nik = nik
        self.__Jenis_Kelamin = Jenis_Kelamin

    # Setter and Getter methods for private attributes
    # set Nama
    def setNama(self, nama):
        self.__nama = nama

    # Get Nama
    def getNama(self):
        return self.__nama

    # Set nik
    def setNIK(self, nik):
        self.__nik = nik

    # Get nik
    def getNik(self):
        return self.__nik

    # Set gender
    def setGender(self, Jenis_Kelamin):
        self.__Jenis_Kelamin = Jenis_Kelamin

    # Get gender
    def getGender(self):
        return self.__Jenis_Kelamin
